cmd/ztest/ztest.c

```diff
@@ -2770,6 +2770,8 @@ ztest_mmp_enable_disable(ztest_ds_t *zd, uint64_t id)
 	spa_config_enter(spa, SCL_CONFIG, FTAG, RW_READER);
 	mutex_enter(&spa->spa_props_lock);
 
+	zfs_multihost_fail_intervals = 0;
```

Comment:
I thought about that, but I don't think it's ever helpful for MMP to suspend the pool within ztest, as the code stands now.

If the pool is suspended due to MMP, it cannot be resumed (unlike a pool suspended for device errors).  Code for that does not currently exist (for either ztest or real pools), but will some day because it would be useful to production users.  However, this breaks the ztest assumption that pools can always be made active again.  This in turn causes ztest failures.

So there are other, better fixes possible, such as allowing pools to be resumed, and making the rest of ztest handle suspended pools better, but those are not going to happen soon.  In the meantime we don't want to waste developers time when ztest fails during automated testing of a patch.